### PR TITLE
Shannon/landing style

### PR DESCRIPTION
**Pull request by shannonchu**
Landing page style with foundation :warning: Still need to change navbar and anchor banner image on landing :warning: 
Some user profile styling

### CODE REVIEW

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -4,45 +4,49 @@
 
 @import "ashby_settings.scss";
 
-.banner_text {
-  font-style: bold;
-  padding-bottom: 150px;
```
Comment (andylouisqin) [inline app/assets/stylesheets/static_pages.scss:9]: I don't remember if I added this in as a quick fix before that one GM, but this isn't sustainable lol. If we use the grid, we can just make sure this text is in the same `row`

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+      <h2><%= @user.full_name %></h2>
+    </div>
+  </div>
 
-<% if @user.photo %>
-  <%= image_tag @user.photo.url(:thumb) %>
-<% end %><br />
+  <div class="nav">
+    <h6>Your Information</h6>
+    <h6>Groups You Lead</h6>
```
Comment (andylouisqin) [inline app/views/users/show.html.erb:13]: Hmm these additions ought to be under another branch/PR right @shannonchu? Probably something like `shannon/user-profile`?

File: app/assets/stylesheets/static_pages.scss [hunk 3]
```diff
@@ -4,45 +4,49 @@
...
-}
-
 .landing {
   text-align: center;
   background: image-url("bg_main.jpg") transparent no-repeat;
   background-size: contain;
-  background-position: 0% -50%;
-  padding: 200px;
-}
+  background-position: 100% -50%;
```
Comment (andylouisqin) [inline app/assets/stylesheets/static_pages.scss:11]: Pretty certain I wrote this in the super hacky push before GM. What I propose is that we can merge this branch in, but I'll make my next focus to clean up the styles for responsiveness. What say you @lilshim?

Comment (hidesminimally) [inline app/assets/stylesheets/static_pages.scss:11]: @andylouisqin @shannonchu Yup. We're going to have to go over every view and make them responsive anyway.

File: app/views/users/show.html.erb (hunk 2 above)
Comment (hidesminimally) [inline app/views/users/show.html.erb:13]: @shannonchu Remember to split up your features in different branches! We can merge this one in, but don't forget next time!

**Comment by andylouisqin** [comment]
Can't quite merge this baby in @shannonchu @lilshim due to merge conflicts. Pull master, then fix the conflicts?

File: app/controllers/user_controller.rb [hunk 4]
```diff
@@ -1,27 +0,0 @@
-class UserController < ApplicationController
```
Comment (andylouisqin) [inline app/controllers/user_controller.rb:1]: Wait @shannonchu did you delete the `user_controller` entirely?

Comment (shannonchu) [inline app/controllers/user_controller.rb:1]: @andylouisqin  Yea, we have a users_controller, so the user_controller was not necessary

Comment (andylouisqin) [inline app/controllers/user_controller.rb:1]: Ok then, just fix those merge conflicts, and I'll merge it right in after!

**Pull request merged**